types/postgraphile/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for postgraphile 4.0
+// Project: https://github.com/postgraphql/postgraphql/tree/postgraphile
```

Comment:
The url says postgraphile, but the README.md there says to `npm install -g postgraphql@next`. We should match the package name from NPM, so this directory should probably be named `postgraphql`.